oak-upgrade/src/main/java/org/apache/jackrabbit/oak/upgrade/cli/node/SegmentAzureFactory.java

```diff
@@ -87,6 +90,10 @@ public Builder containerName(String containerName) {
             this.containerName = containerName;
             return this;
         }
+        public Builder sasToken(String sasToken) {
```

Comment:
Is it necessary to explicitly set ```sasToken```? I have impression that it can be the part of ```connectionString```.
https://docs.microsoft.com/en-us/azure/storage/common/storage-configure-connection-string#create-a-connection-string-using-a-shared-access-signature

Reply:
I think it can be useful in the case where an URL of form `https://path/container?sasToken` is used instead of a connection string.

Reply:
Ok, I see that this is set after parsing the connection in org.apache.jackrabbit.oak.upgrade.cli.parser.StoreType.